lib/namespace.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const asyncHooks = require('async_hooks');
```

Comment:
`async_hooks` was only added to core in Node 8 and above. This is going to cause issues on Node 6